chain/chain/src/runtime/tests.rs

```diff
@@ -1061,6 +1056,7 @@ fn test_challenges() {
 
 /// Test that in case of a double sign, not all stake is slashed if the double signed stake is
 /// less than 33% and all stake is slashed if the stake is more than 33%
+#[ignore]
```

Comment:
Should we write a quick reason for ignore as well? 
`#[ignore = "Ignoring challenge and slashing related tests"]`